xcdat/dataset.py

```diff
@@ -308,16 +313,33 @@ def decode_non_cf_time(dataset: xr.Dataset) -> xr.Dataset:
     time_bounds = ds.get(time.attrs.get("bounds"), None)
     units_attr = time.attrs.get("units")
 
-    # If the time units cannot be split into a unit and reference date, it
-    # cannot be decoded so the original dateset is returned.
+    # Default to stanadard if no calendar is set per:
+    # "If the calendar kwarg is set to a blank string (‘’) or None
+    # (the default is ‘standard’) the instance will not be calendar-aware and
+    # some methods will not work".
+    # Source: https://unidata.github.io/cftime/api.html#cftime.datetime
+    calendar_attr = time.attrs.get("calendar", "standard")
```

Comment:
The calendar attribute is now considered when creating `cftime.datetime` objects.